### PR TITLE
Rollback command support for max history

### LINKED ISSUE #8370: Rollback command support for max history
Ref #8367

after https://github.com/helm/helm/pull/8369

### PR DESCRIPTION
**Pull request by wawa0210**
**What this PR does / why we need it**:
Rollback command support for max history 

Fix https://github.com/helm/helm/issues/8370

### CODE REVIEW

**Comment by wawa0210** [comment]
@hickeyma 
Look forward to your comments

**Review by liuming-dev — COMMENTED** [review]
Do you mind adding a unit test?

**Review by marckhouzam — COMMENTED** [review]
This is nice. Thank you.

File: pkg/action/rollback.go [hunk 1]
```diff
@@ -42,6 +42,8 @@ type Rollback struct {
 	Recreate      bool // will (if true) recreate pods after a rollback.
 	Force         bool // will (if true) force resource upgrade through uninstall/recreate if needed
 	CleanupOnFail bool
+	// MaxHistory limits the maximum number of revisions saved per release
+	MaxHistory int
```
Comment (marckhouzam) [inline pkg/action/rollback.go:46]: I'm on my phone so I could be wrong, but this doesn't look like it's properly formatted.  Could you run `make test-style` to confirm?

Comment (wawa0210) [reply pkg/action/rollback.go]: Thx,I put the comment at the end of the line so it looks normal

**Review by hickeyma — APPROVED** [review]
LGMT, thanks @wawa0210

**Pull request merged**